management/backup.py

```diff
@@ -283,7 +278,7 @@ def service_command(service, command, quit=None):
 			if quit:
 				sys.exit(code)
 
-	service_command("php7.2-fpm", "stop", quit=True)
+	service_command("php8.0-fpm", "stop", quit=True)
```

Comment:
There are some instances that have ...php8.0... and some that have php$PHP_VER, should this be unified to also use `phpPHP_VER`?